command/test/type/boolean_test.ts

```diff
@@ -15,6 +15,14 @@ const cmd2 = new Command()
   .option("-f, --flag [value:boolean]", "description ...")
   .action(() => {});
 
+const cmd3 = new Command()
+  .throwErrors()
+  .name("test-command")
+  .option("-f, --check", "description ...")
+  .option("-f, --no-check", "description ...")
```

Comment:
the `-f` option is used twice.
```suggestion
  .option("--no-check", "description ...")
```

Reply:
hmm weird I get green tests: `ok | 663 passed (23 steps) | 0 failed (6s)`, with:

Reply:
@c4spar should be fixed now.